### DISCUSSION TITLE
Clarification on serving bundled app via HTTP

### DISCUSSION THREAD

**Discussion by gzuuus** [wailsapp/wails#3812] [Q&A]
Hi Wails community,
I've been exploring Wails as a way to bundle a frontend with my Go backend, and I'm excited about the possibilities. However, I have a question regarding the use case of serving the bundled app via HTTP to make it accessible from the internet.
From the documentation, I understand that Wails is primarily designed for creating desktop applications. But I'm wondering if it's possible or valid to serve the bundled app through an HTTP server, making it accessible from the internet.
Is this a supported use case, or are there any limitations or concerns that I should be aware of? I'd love to hear from the maintainers or experienced users about their thoughts on this.

**Comment by leaanthony** [comment]
This is something we are looking at for v3 👍
